.github/workflows/release-test.yml

```diff
@@ -13,8 +13,8 @@ on:
   push
```

Comment:
I think this is a waste of CI resources, just look how many commits you have in this PR alone. Let's have this as a manually triggered job in the release checklist.

Reply:
I'm happy to remove this CI, but I don't care as much about CI resources that run on main or on repository branches.

I already added it as a manually triggered job in the release checklist.

Reply:
Anything actionable?